azuredeploy-ddosprotection.parameters.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
I don't think we want this file, at least not in its current form. Our parameters files are for specific example workloads and show different VM scales etc. We don't carry examples for individual features. Compare this file to https://github.com/Azure/Moodle/blob/master/azuredeploy.parameters.json, for example.

Rather than this file I'd much rather see an update to the README.md and https://github.com/Azure/Moodle/blob/master/docs/Parameters.md (note the latter is auto-generated by a script described at the top of the document).